Tests/AdyenTests/Adyen Tests/Components/Gift Card/GiftCardComponentTests.swift

```diff
@@ -236,44 +233,6 @@ class GiftCardComponentTests: XCTestCase {
         waitForExpectations(timeout: 10, handler: nil)
     }
 
-    func testMissingPaymentObject() throws {
```

Comment:
why is this removed? payment object is still optional, right?

Reply:
no, not any more :)
DropIn will emit console log / assert failure, but component itself just expects the amount (like ApplePay)

Reply:
why keep it optional then?

Reply:
It [looks like this](https://github.com/Adyen/adyen-ios/pull/940/files#diff-516c91ca2ef64f84384962f06bd375a2c91af3415467d52c9129f6cae0fa3decR55-R58):

```swift 
public convenience init(paymentMethod: GiftCardPaymentMethod,
                            context: AdyenContext,
                            amount: Amount,
                            style: FormComponentStyle = FormComponentStyle())
```